src/components/navigation/breadcrumb/Breadcrumb.tsx

```diff
@@ -5,19 +5,20 @@ import type { AsElementLink } from '../../../util/types/LinkTypes';
 
 type Item = React.FC<AsElementLink<HTMLAnchorElement>>;
 
-const Item: Item = ({
-  className, children, asElement: Component = 'a', ...rest
-}) => (
+const Item: Item = ({ className, children, asElement: Component = 'a', ...rest }) => (
   <li className="nhsuk-breadcrumb__item">
     <Component className={classNames('nhsuk-breadcrumb__link', className)} {...rest}>
       {children}
     </Component>
   </li>
 );
 
-const Back: Item = ({ className, asElement: Component = 'a', ...rest }) => (
+const Back: Item = ({ className, children, asElement: Component = 'a', ...rest }) => (
   <p className={classNames('nhsuk-breadcrumb__back', className)}>
-    <Component className="nhsuk-breadcrumb__backlink" {...rest} />
+    <Component className="nhsuk-breadcrumb__backlink" {...rest}>
+      <span className="nhsuk-u-visually-hidden">Back to &nbsp;</span>
```

Comment:
be worth making this a prop and defaulting

Reply:
Yes, done